### PR TITLE
Add Aria2 build

### PR DESCRIPTION
**Pull request by jreese**
_Motivation:_ Adds build script for `aria2c` download manager.
_Linked issues:_ #3269 

Only tested on DSM 6.1 on the ds916+.

### Checklist
- [] Build rule `all-supported` completed successfully
- [] Package upgrade completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

**Comment by ymartin59** [comment]
Thanks for this work. As a small CLI tool, I suggest to include it in one of planned multi-tool packages, see #2261
Such a PR to `dsm6` will not be merged, as this branch is reserved to support for DSM 6.x. Please rebase your branch to `master`.

File: spk/aria2/Makefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+DISPLAY_NAME = aria2
+STARTABLE = no
+BETA = 0
+CHANGELOG = Initial version
+
+HOMEPAGE   = http://aria2.github.io
+LICENSE    = GPLv2/GPLv3
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
```
Comment (ymartin59) [inline spk/aria2/Makefile]: Please remove `SSS_SCRIPT` and `dsm-control.sh` file as application does not run as a service

File: spk/aria2/Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+SPK_NAME = aria2
+SPK_VERS = 1.31.0
+SPK_REV = 1
+SPK_ICON = src/aria2.png
```
Comment (ymartin59) [inline spk/aria2/Makefile]: What I see on screen is `tmux` and `top`... If aria is included in a CLI multi-tool package, no need for an icon by the way.

**Comment by cytec** [comment]
@ymartin59 not so sure about the CLI Multipackage for that one. It's a pretty solid/big download client... I'd compare it more to something like rutorrent or something like that than "a small cli tool" so i think it has a right for his own SPK. Maybe we could even include https://github.com/ziahamza/webui-aria2 which seems to be a pretty solid webui for aria2 this days...

**Comment by ymartin59** [comment]
@cytec OK I agree with you. It may be interesting to include webui in same package when NodeJS cross-compilation support #2793 will be available for packages.

**Pull request closed without merging**